pkg/payload/payload.go

```diff
@@ -412,3 +437,8 @@ func loadImageReferences(releaseDir string) (*imagev1.ImageStream, error) {
 	}
 	return nil, fmt.Errorf("%s is a %T, not a v1 ImageStream", imageReferencesFile, imageRefObj)
 }
+
+// isHeterogeneousArchitecture returns true if arch is set to heterogeneousArchitectureID indicating a heterogeneous architecture.
+func isHeterogeneousArchitecture(arch string) bool {
+	return arch == heterogeneousArchitectureID
```

Comment:
nit: at the moment, the `==` seems straightforward enough that I'd rather inline it and drop `isHeterogeneousArchitecture`.  But I don't feel all that strongly about it, so feel free to ignore me if you prefer it the way you have it.

Reply:
Prefer to leave it and just move on since it has to be touched again to fully implement heter.

Reply:
done